shell/platform/darwin/ios/framework/Source/FlutterTextureRegistryRelay.h

```diff
@@ -22,7 +22,7 @@ FLUTTER_DARWIN_EXPORT
 /**
  * A weak reference to a FlutterEngine that will be passed texture registration.
  */
-@property(nonatomic, assign) NSObject<FlutterTextureRegistry>* parent;
+@property(nonatomic, weak) NSObject<FlutterTextureRegistry>* parent;
```

Comment:
is there a place where we manually `nil` out this `parent`? 

if we have it, we can remove it now; If we don't already have it, we may have dangling pointer problem which is fixed by this PR.

Reply:
The parent here is the FlutterEngine:
https://github.com/flutter/engine/blob/6f8ee9ffd9fae18b10a4ca3888f1ac115d8cb300/shell/platform/darwin/ios/framework/Source/FlutterEngine.mm#L310
See also the last time we looked at this: https://github.com/flutter/engine/pull/37666

I'll leave this for now, maybe we can do a sweep of these when I'm done ARC-ifying.